src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -24,6 +24,13 @@ public void SendUndoToTutorial(TutorialState tutorial)
         tutorial.AutoEvoPrediction.EditorAutoEvoPredictionPanel = autoEvoPredictionPanel;
     }
 
+    public void SendRedoToTutorial(TutorialState tutorial)
```

Comment:
Maybe it would make more sense to combine this method with `SendUndoToTutorial` (and rename it something like `SendUndoRedoToTutorial`). Also it seems you've accidentally duplicated the auto evo prediction sending in this method.